smartsheet-csharp-sdk/main/Smartsheet/Api/SheetResources.cs

```diff
@@ -91,6 +91,9 @@ public interface SheetResources
         /// <param name="columnIds"> used to specify the optional objects to include. </param>
         /// <param name="pageSize"> used to specify the optional objects to include. </param>
         /// <param name="page"> used to specify the optional objects to include. </param>
+        /// <param name="rowsModifiedSince">FIXME: WHAT IS THIS PARAM</param>
```

Comment:
what should I include here?

Reply:
I might consider pulling in mainline. I don't see these on mainline.... ?

Reply:
I see the yellow lines under the parameters to GetSheet. But I can pull them for now.
<img width="1000" alt="image" src="https://github.com/user-attachments/assets/1a43378c-261d-4329-a242-da543a4feb8d">